src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs

```diff
@@ -220,6 +226,50 @@ public static void WriteRequestCreation(CodeWriter writer, RestClientMethod clie
             writer.Line();
         }
 
+        private static void WriteUriCreation(CodeWriter writer, RestClientMethod clientMethod, string methodAccessibility, ClientFields? fields, IReadOnlyList<Parameter>? clientParameters = null)
+        {
+            using var methodScope = writer.AmbientScope();
+            var methodName = CreateRequestUriMethodName($"{clientMethod.Name}");
+            writer.Append($"{methodAccessibility} {typeof(RequestUriBuilder)} {methodName}(");
+            foreach (Parameter clientParameter in clientMethod.Parameters)
+            {
+                writer.Append($"{clientParameter.Type} {clientParameter.Name:D},");
```

Comment:
We should use the expressions here instead of raw strings

Reply:
Postpone this. If we can find a way to get Http method for fake LRO, we don't need this change.

Reply:
change reverted

Reply:
Need to add this back, and this is just part of the [existing code](https://github.com/Azure/autorest.csharp/blob/49a5dfc9d7634bb901d53f20b25ceb4239201456/src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs#L72). Rewrite them into expressions takes more effort than expected. I would prefer to defer this until we convert the entire code into expression.

Reply:
create an issue for this

Reply:
https://github.com/Azure/autorest.csharp/issues/4335